[WebGPU] Update WebGPU.h to latest revision

mwyrzykowski opened the pull request:
#### 26f539228dd0a3ee85e38e886753c41955e93971
<pre>
[WebGPU] Update WebGPU.h to latest revision
<a href="https://bugs.webkit.org/show_bug.cgi?id=261511">https://bugs.webkit.org/show_bug.cgi?id=261511</a>&gt;
&lt;radar://115430040&gt;

Reviewed by Dan Glastonbury.

Update WebGPU paths with some changes to the specification over the past few months.

Largely specification conformance change.

* Source/WebCore/CMakeLists.txt:
* Source/WebCore/DerivedSources-input.xcfilelist:
* Source/WebCore/DerivedSources-output.xcfilelist:
* Source/WebCore/DerivedSources.make:
* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/WebGPU/GPUCanvasAlphaMode.h: Renamed from Source/WebCore/Modules/WebGPU/GPUCanvasCompositingAlphaMode.h.
(WebCore::convertToBacking):
* Source/WebCore/Modules/WebGPU/GPUCanvasAlphaMode.idl: Renamed from Source/WebCore/Modules/WebGPU/GPUCanvasCompositingAlphaMode.idl.
* Source/WebCore/Modules/WebGPU/GPUCanvasConfiguration.h:
(WebCore::GPUCanvasConfiguration::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPUCanvasConfiguration.idl:
* Source/WebCore/Modules/WebGPU/GPUComputePassDescriptor.h:
(WebCore::GPUComputePassDescriptor::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPUComputePassDescriptor.idl:
* Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrite.h: Removed.
* Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrites.h: Copied from Source/WebCore/Modules/WebGPU/GPUComputePassDescriptor.h.
(WebCore::GPUComputePassTimestampWrites::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrites.idl: Renamed from Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrite.idl.
* Source/WebCore/Modules/WebGPU/GPUDepthStencilState.idl:
* Source/WebCore/Modules/WebGPU/GPUDevice.cpp:
(WebCore::GPUDevice::createTexture):
* Source/WebCore/Modules/WebGPU/GPUImageCopyExternalImage.h:
* Source/WebCore/Modules/WebGPU/GPUImageCopyExternalImage.idl:
* Source/WebCore/Modules/WebGPU/GPUIntegralTypes.h:
* Source/WebCore/Modules/WebGPU/GPUPresentationContext.cpp:
(WebCore::GPUPresentationContext::getCurrentTexture):
* Source/WebCore/Modules/WebGPU/GPUQueue.cpp:
(WebCore::imageBufferForSource):
(WebCore::toPixelFormat):
* Source/WebCore/Modules/WebGPU/GPURenderPassDescriptor.h:
(WebCore::GPURenderPassDescriptor::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPURenderPassDescriptor.idl:
* Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrite.h: Removed.
* Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrites.h: Copied from Source/WebCore/Modules/WebGPU/InternalAPI/WebGPURenderPassDescriptor.h.
(WebCore::GPURenderPassTimestampWrites::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrites.idl: Renamed from Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrite.idl.
* Source/WebCore/Modules/WebGPU/GPUShaderModule.cpp:
(WebCore::GPUShaderModule::getCompilationInfo):
(WebCore::GPUShaderModule::compilationInfo): Deleted.
* Source/WebCore/Modules/WebGPU/GPUShaderModule.h:
* Source/WebCore/Modules/WebGPU/GPUShaderModule.idl:
* Source/WebCore/Modules/WebGPU/GPUSupportedLimits.cpp:
(WebCore::GPUSupportedLimits::maxBindGroupsPlusVertexBuffers const):
* Source/WebCore/Modules/WebGPU/GPUSupportedLimits.h:
* Source/WebCore/Modules/WebGPU/GPUSupportedLimits.idl:
* Source/WebCore/Modules/WebGPU/GPUTexture.cpp:
(WebCore::getDimension):
(WebCore::GPUTexture::GPUTexture):
(WebCore::GPUTexture::width const):
(WebCore::GPUTexture::height const):
(WebCore::GPUTexture::depthOrArrayLayers const):
(WebCore::GPUTexture::mipLevelCount const):
(WebCore::GPUTexture::sampleCount const):
(WebCore::GPUTexture::dimension const):
(WebCore::GPUTexture::usage const):
* Source/WebCore/Modules/WebGPU/GPUTexture.h:
(WebCore::GPUTexture::create):
(WebCore::GPUTexture::GPUTexture): Deleted.
* Source/WebCore/Modules/WebGPU/GPUTexture.idl:
* Source/WebCore/Modules/WebGPU/GPUTextureFormat.h:
(WebCore::convertToBacking):
* Source/WebCore/Modules/WebGPU/GPUTextureFormat.idl:
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUAdapterImpl.cpp:
(WebCore::WebGPU::supportedLimits):
(WebCore::WebGPU::AdapterImpl::requestDevice):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUCommandEncoderImpl.cpp:
(WebCore::WebGPU::CommandEncoderImpl::beginRenderPass):
(WebCore::WebGPU::CommandEncoderImpl::beginComputePass):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUConvertToBackingContext.cpp:
(WebCore::WebGPU::ConvertToBackingContext::convertToBacking):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUImpl.cpp:
(WebCore::WebGPU::GPUImpl::requestAdapter):
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUCanvasAlphaMode.h: Renamed from Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUCanvasCompositingAlphaMode.h.
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUCanvasConfiguration.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUComputePassDescriptor.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUComputePassTimestampWrites.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPURenderPassDescriptor.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPURenderPassTimestampWrites.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUSupportedLimits.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUTextureFormat.h:
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/html/canvas/GPUCanvasContextCocoa.h:
* Source/WebCore/html/canvas/GPUCanvasContextCocoa.mm:
(WebCore::GPUCanvasContextCocoa::configure):
* Source/WebGPU/WebGPU/Adapter.mm:
(WebGPU::Adapter::requestDevice):
(wgpuAdapterGetLimits):
(wgpuAdapterHasFeature):
* Source/WebGPU/WebGPU/Buffer.mm:
(wgpuBufferGetUsage):
* Source/WebGPU/WebGPU/ComputePassEncoder.mm:
(wgpuComputePassEncoderSetBindGroup):
* Source/WebGPU/WebGPU/Device.mm:
(wgpuDeviceGetLimits):
(wgpuDeviceHasFeature):
* Source/WebGPU/WebGPU/HardwareCapabilities.mm:
(WebGPU::apple3):
(WebGPU::apple4):
(WebGPU::apple5):
(WebGPU::apple6):
(WebGPU::apple7):
(WebGPU::mac2):
(WebGPU::mergeLimits):
(WebGPU::defaultLimits):
* Source/WebGPU/WebGPU/Queue.mm:
(wgpuQueueSubmit):
* Source/WebGPU/WebGPU/RenderBundleEncoder.mm:
(wgpuRenderBundleEncoderSetBindGroup):
* Source/WebGPU/WebGPU/RenderPassEncoder.mm:
(wgpuRenderPassEncoderExecuteBundles):
(wgpuRenderPassEncoderSetBindGroup):
* Source/WebGPU/WebGPU/ShaderModule.mm:
(WebGPU::convertBindingLayout):
* Source/WebGPU/WebGPU/Texture.mm:
(wgpuTextureGetUsage):
* Source/WebGPU/WebGPU/WebGPU.h:
* Source/WebGPU/WebGPU/WebGPUExt.h:
* Source/WebKit/GPUProcess/graphics/WebGPU/RemoteAdapter.cpp:
(WebKit::RemoteAdapter::requestDevice):
* Source/WebKit/GPUProcess/graphics/WebGPU/RemoteGPU.cpp:
(WebKit::RemoteGPU::requestAdapter):
* Source/WebKit/Scripts/webkit/messages.py:
(headers_for_type):
* Source/WebKit/Shared/WebGPU/WebGPUCanvasConfiguration.h:
* Source/WebKit/Shared/WebGPU/WebGPUCanvasConfiguration.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassDescriptor.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPUComputePassDescriptor.h:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassDescriptor.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassTimestampWrites.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPUComputePassTimestampWrites.h:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassTimestampWrites.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUConvertFromBackingContext.h:
* Source/WebKit/Shared/WebGPU/WebGPUConvertToBackingContext.h:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassDescriptor.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPURenderPassDescriptor.h:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassDescriptor.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassTimestampWrites.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPURenderPassTimestampWrites.h:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassTimestampWrites.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUSupportedLimits.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPUSupportedLimits.h:
* Source/WebKit/Shared/WebGPU/WebGPUSupportedLimits.serialization.in:
* Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteAdapterProxy.cpp:
(WebKit::WebGPU::RemoteAdapterProxy::requestDevice):
* Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteGPUProxy.cpp:
(WebKit::RemoteGPUProxy::requestAdapter):

Canonical link: <a href="https://commits.webkit.org/268163@main">https://commits.webkit.org/268163@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/57adf838394c5cf9a64fa9147cbd9f3a4c00198f

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/18379 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/26/builds/18717 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/19310 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/20228 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/17207 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/18575 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/23/builds/22023 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/16/builds/18866 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/19120 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/18603 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/15/builds/18799 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/16010 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/21107 "Built successfully") | 
| [✅ 🧪 webkitpy](https://ews-build.webkit.org/#/builders/6/builds/19041 "Passed tests") | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/39/builds/16022 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/16767 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/23245 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/17043 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/16937 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/21132 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/7/builds/17507 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/17/builds/14854 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/30/builds/16590 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/4488 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/4/builds/20956 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/31/builds/17346 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

djg commented on Source/WebGPU/WebGPU/Adapter.mm:
```diff
@@ -147,7 +147,7 @@ size_t wgpuAdapterEnumerateFeatures(WGPUAdapter adapter, WGPUFeatureName* featur
 }
 
-bool wgpuAdapterGetLimits(WGPUAdapter adapter, WGPUSupportedLimits* limits)
+WGPUBool wgpuAdapterGetLimits(WGPUAdapter adapter, WGPUSupportedLimits* limits)
```
What other kind of bool is there?

mwyrzykowski replied:
They typedef’ed this as an int, not sure why

djg reviewed (commented):
:shipit:

djg reviewed (approved):
:shipit:

mwyrzykowski committed:
[WebGPU] Update WebGPU.h to latest revision
https://bugs.webkit.org/show_bug.cgi?id=261511>
<radar://115430040>

Reviewed by Dan Glastonbury.

Update WebGPU paths with some changes to the specification over the past few months.

Largely specification conformance change.

* Source/WebCore/CMakeLists.txt:
* Source/WebCore/DerivedSources-input.xcfilelist:
* Source/WebCore/DerivedSources-output.xcfilelist:
* Source/WebCore/DerivedSources.make:
* Source/WebCore/Headers.cmake:
* Source/WebCore/Modules/WebGPU/GPUCanvasAlphaMode.h: Renamed from Source/WebCore/Modules/WebGPU/GPUCanvasCompositingAlphaMode.h.
(WebCore::convertToBacking):
* Source/WebCore/Modules/WebGPU/GPUCanvasAlphaMode.idl: Renamed from Source/WebCore/Modules/WebGPU/GPUCanvasCompositingAlphaMode.idl.
* Source/WebCore/Modules/WebGPU/GPUCanvasConfiguration.h:
(WebCore::GPUCanvasConfiguration::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPUCanvasConfiguration.idl:
* Source/WebCore/Modules/WebGPU/GPUComputePassDescriptor.h:
(WebCore::GPUComputePassDescriptor::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPUComputePassDescriptor.idl:
* Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrite.h: Removed.
* Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrites.h: Copied from Source/WebCore/Modules/WebGPU/GPUComputePassDescriptor.h.
(WebCore::GPUComputePassTimestampWrites::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrites.idl: Renamed from Source/WebCore/Modules/WebGPU/GPUComputePassTimestampWrite.idl.
* Source/WebCore/Modules/WebGPU/GPUDepthStencilState.idl:
* Source/WebCore/Modules/WebGPU/GPUDevice.cpp:
(WebCore::GPUDevice::createTexture):
* Source/WebCore/Modules/WebGPU/GPUImageCopyExternalImage.h:
* Source/WebCore/Modules/WebGPU/GPUImageCopyExternalImage.idl:
* Source/WebCore/Modules/WebGPU/GPUIntegralTypes.h:
* Source/WebCore/Modules/WebGPU/GPUPresentationContext.cpp:
(WebCore::GPUPresentationContext::getCurrentTexture):
* Source/WebCore/Modules/WebGPU/GPUQueue.cpp:
(WebCore::imageBufferForSource):
(WebCore::toPixelFormat):
* Source/WebCore/Modules/WebGPU/GPURenderPassDescriptor.h:
(WebCore::GPURenderPassDescriptor::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPURenderPassDescriptor.idl:
* Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrite.h: Removed.
* Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrites.h: Copied from Source/WebCore/Modules/WebGPU/InternalAPI/WebGPURenderPassDescriptor.h.
(WebCore::GPURenderPassTimestampWrites::convertToBacking const):
* Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrites.idl: Renamed from Source/WebCore/Modules/WebGPU/GPURenderPassTimestampWrite.idl.
* Source/WebCore/Modules/WebGPU/GPUShaderModule.cpp:
(WebCore::GPUShaderModule::getCompilationInfo):
(WebCore::GPUShaderModule::compilationInfo): Deleted.
* Source/WebCore/Modules/WebGPU/GPUShaderModule.h:
* Source/WebCore/Modules/WebGPU/GPUShaderModule.idl:
* Source/WebCore/Modules/WebGPU/GPUSupportedLimits.cpp:
(WebCore::GPUSupportedLimits::maxBindGroupsPlusVertexBuffers const):
* Source/WebCore/Modules/WebGPU/GPUSupportedLimits.h:
* Source/WebCore/Modules/WebGPU/GPUSupportedLimits.idl:
* Source/WebCore/Modules/WebGPU/GPUTexture.cpp:
(WebCore::getDimension):
(WebCore::GPUTexture::GPUTexture):
(WebCore::GPUTexture::width const):
(WebCore::GPUTexture::height const):
(WebCore::GPUTexture::depthOrArrayLayers const):
(WebCore::GPUTexture::mipLevelCount const):
(WebCore::GPUTexture::sampleCount const):
(WebCore::GPUTexture::dimension const):
(WebCore::GPUTexture::usage const):
* Source/WebCore/Modules/WebGPU/GPUTexture.h:
(WebCore::GPUTexture::create):
(WebCore::GPUTexture::GPUTexture): Deleted.
* Source/WebCore/Modules/WebGPU/GPUTexture.idl:
* Source/WebCore/Modules/WebGPU/GPUTextureFormat.h:
(WebCore::convertToBacking):
* Source/WebCore/Modules/WebGPU/GPUTextureFormat.idl:
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUAdapterImpl.cpp:
(WebCore::WebGPU::supportedLimits):
(WebCore::WebGPU::AdapterImpl::requestDevice):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUCommandEncoderImpl.cpp:
(WebCore::WebGPU::CommandEncoderImpl::beginRenderPass):
(WebCore::WebGPU::CommandEncoderImpl::beginComputePass):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUConvertToBackingContext.cpp:
(WebCore::WebGPU::ConvertToBackingContext::convertToBacking):
* Source/WebCore/Modules/WebGPU/Implementation/WebGPUImpl.cpp:
(WebCore::WebGPU::GPUImpl::requestAdapter):
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUCanvasAlphaMode.h: Renamed from Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUCanvasCompositingAlphaMode.h.
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUCanvasConfiguration.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUComputePassDescriptor.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUComputePassTimestampWrites.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPURenderPassDescriptor.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPURenderPassTimestampWrites.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUSupportedLimits.h:
* Source/WebCore/Modules/WebGPU/InternalAPI/WebGPUTextureFormat.h:
* Source/WebCore/Sources.txt:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/html/canvas/GPUCanvasContextCocoa.h:
* Source/WebCore/html/canvas/GPUCanvasContextCocoa.mm:
(WebCore::GPUCanvasContextCocoa::configure):
* Source/WebGPU/WebGPU/Adapter.mm:
(WebGPU::Adapter::requestDevice):
(wgpuAdapterGetLimits):
(wgpuAdapterHasFeature):
* Source/WebGPU/WebGPU/Buffer.mm:
(wgpuBufferGetUsage):
* Source/WebGPU/WebGPU/ComputePassEncoder.mm:
(wgpuComputePassEncoderSetBindGroup):
* Source/WebGPU/WebGPU/Device.mm:
(wgpuDeviceGetLimits):
(wgpuDeviceHasFeature):
* Source/WebGPU/WebGPU/HardwareCapabilities.mm:
(WebGPU::apple3):
(WebGPU::apple4):
(WebGPU::apple5):
(WebGPU::apple6):
(WebGPU::apple7):
(WebGPU::mac2):
(WebGPU::mergeLimits):
(WebGPU::defaultLimits):
* Source/WebGPU/WebGPU/Queue.mm:
(wgpuQueueSubmit):
* Source/WebGPU/WebGPU/RenderBundleEncoder.mm:
(wgpuRenderBundleEncoderSetBindGroup):
* Source/WebGPU/WebGPU/RenderPassEncoder.mm:
(wgpuRenderPassEncoderExecuteBundles):
(wgpuRenderPassEncoderSetBindGroup):
* Source/WebGPU/WebGPU/ShaderModule.mm:
(WebGPU::convertBindingLayout):
* Source/WebGPU/WebGPU/Texture.mm:
(wgpuTextureGetUsage):
* Source/WebGPU/WebGPU/WebGPU.h:
* Source/WebGPU/WebGPU/WebGPUExt.h:
* Source/WebKit/GPUProcess/graphics/WebGPU/RemoteAdapter.cpp:
(WebKit::RemoteAdapter::requestDevice):
* Source/WebKit/GPUProcess/graphics/WebGPU/RemoteGPU.cpp:
(WebKit::RemoteGPU::requestAdapter):
* Source/WebKit/Scripts/webkit/messages.py:
(headers_for_type):
* Source/WebKit/Shared/WebGPU/WebGPUCanvasConfiguration.h:
* Source/WebKit/Shared/WebGPU/WebGPUCanvasConfiguration.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassDescriptor.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPUComputePassDescriptor.h:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassDescriptor.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassTimestampWrites.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPUComputePassTimestampWrites.h:
* Source/WebKit/Shared/WebGPU/WebGPUComputePassTimestampWrites.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUConvertFromBackingContext.h:
* Source/WebKit/Shared/WebGPU/WebGPUConvertToBackingContext.h:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassDescriptor.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPURenderPassDescriptor.h:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassDescriptor.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassTimestampWrites.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPURenderPassTimestampWrites.h:
* Source/WebKit/Shared/WebGPU/WebGPURenderPassTimestampWrites.serialization.in:
* Source/WebKit/Shared/WebGPU/WebGPUSupportedLimits.cpp:
(WebKit::WebGPU::ConvertToBackingContext::convertToBacking):
(WebKit::WebGPU::ConvertFromBackingContext::convertFromBacking):
* Source/WebKit/Shared/WebGPU/WebGPUSupportedLimits.h:
* Source/WebKit/Shared/WebGPU/WebGPUSupportedLimits.serialization.in:
* Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteAdapterProxy.cpp:
(WebKit::WebGPU::RemoteAdapterProxy::requestDevice):
* Source/WebKit/WebProcess/GPU/graphics/WebGPU/RemoteGPUProxy.cpp:
(WebKit::RemoteGPUProxy::requestAdapter):

Canonical link: https://commits.webkit.org/268163@main
